src/pcapi/settings.py

```diff
@@ -118,3 +118,9 @@
 
 # JWT
 JWT_SECRET_KEY = os.environ.get("JWT_SECRET_KEY")
+
+
+# TITELIVE
+FTP_TITELIVE_URI = os.environ.get("FTP_TITELIVE_URI")
+FTP_TITELIVE_USER = os.environ.get("FTP_TITELIVE_USER")
+FTP_TITELIVE_PWD = os.environ.get("FTP_TITELIVE_PWD")
```

Comment:
Micro-détail : je proposerais bien de préfixer par `TITELIVE_`, comme suit :

```suggestion
TITELIVE_FTP_URI = os.environ.get("FTP_TITELIVE_URI")
TITELIVE_FTP_USER = os.environ.get("FTP_TITELIVE_USER")
TITELIVE_FTP_PWD = os.environ.get("FTP_TITELIVE_PWD")
```

Évidemment, le nom du setting n'est plus le même que celui de la variable d'environnement, mais ça pourrait se gérer dans un second temps.

Reply:
J'aime bien ta proposition et tant que l'on est sur le sujet, autant le faire.